test/recipes/06-test-rdrand.t

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/perl
+
+# Copyright 2018-2018 The OpenSSL Project Authors. All Rights Reserved.
+# 
+# Licensed under the OpenSSL license (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+use strict;
+
+use OpenSSL::Test;              # get 'plan'
+use OpenSSL::Test::Simple;
+use OpenSSL::Test::Utils;
+
+setup("test_rdrand_sanity");
+
+plan skip_all => "This test is unsupported if static builds are not enabled"
+    if disabled("static");
```

Comment:
Probably you want a test more like this one to avoid the Appveyor failure:
````
plan skip_all => "This test is unsupported in a shared library build on Windows"
    if $^O eq 'MSWin32' && !disabled("shared");
````

Reply:
Ah... that's a bit of an unfortunate platform-specific hack. This is something where I'd actually specifically like it get regression-tested on windows, due to the difference in calling convention between Windows and SysV. Is there an appveyor or other CI test that builds and tests in a windows static library configuration? I see there's a "plain" configuration but that seems to have completed suspiciously quickly.

Reply:
Try adding "[extended tests]" in a line on its own in your commit message.

Reply:
Ah, didn't see this hint. I've tried a different approach to make the tests work with the shared library builds.